daemon/src/model.rs

```diff
@@ -27,6 +32,7 @@ impl Usd {
         Ok(Usd(result))
     }
 
+    // TODO: Usd * Usd = Usd^2 not Usd !!!
```

Comment:
What do you mean with this todo?

Reply:
The function signature claims that multiplying `Usd` with `Usd` gives you `Usd` but that is mathematically not correct. The correct result would be `Usd`^2.

You cannot just ignore units when doing arithmetic.

The argument of this function should likely be changed from `Usd` to a scalar like `Decimal` or `u32` or something.

Reply:
@DeliciousHair is going to fix this with #365.